sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -412,6 +412,9 @@ private boolean doesArgsHaveThrowable(Object... args) {
      * @return The updated logMessage.
      */
     private static String sanitizeLogMessageInput(String logMessage) {
+        if (CoreUtils.isNullOrEmpty(logMessage)) {
```

Comment:
Did `null` or `""` cause an exception or is this a performance enhancement?

Reply:
Yeah, this caused an exception in `GeoArrayTests` - NPE was thrown instead of IndexOutOfBoundsException.